doc/install.rst

```diff
@@ -362,13 +365,37 @@ CMake ``ARB_WITH_PYTHON`` option:
 
 .. code-block:: bash
 
-    cmake -ARB_WITH_PYTHON=ON
+    cmake -DARB_WITH_PYTHON=ON
+
+By default ``ARB_WITH_PYTHON=OFF``. When this option is turned on, a Python module called :py:mod:`arbor` is built.
+
+A specific version of Python can be set when configuring with CMake using the
+``PYTHON_EXECUTABLE`` variable. For example, to use Python 3.7 installed on a Linux
+system with the executable in ``/usr/bin/python3.7``:
+
+.. code-block:: bash
+
+    cmake .. -DARB_WITH_PYTHON=ON -DPYTHON_EXECUTABLE=/usr/bin/python3.7
+
+By default the Python module will be installed in the standard ``CMAKE_INSTALL_PREFIX``
+location. To install the module in a different location, for example as a
+user module or in a virtual environment, set ``ARB_PYTHON_PREFIX``.
+For example, the CMake configuration for targetting Python 3.8 and install as a
+user site package might look like the following:
+
+.. code-block:: bash
+
+    cmake .. -DARB_WITH_PYTHON=ON                   \
+             -DARB_PYTHON_PREFIX=${HOME}/.local/lib \
+             -DPYTHON_EXECUTABLE=/user/bin/python3.8
 
-By default ``ARB_WITH_PYTHON=OFF``. When this option is turned on, a python module called :py:mod:`arbor` is built.
+On the target LINUX system, the Arbor package was installed in
+``/home/$USER/.local/lib/python3.8/site-packages``.
```

Comment:
On Windows the path should be `{prefix}\Lib\site-packages` according to https://docs.python.org/3/library/site.html. Some sources say capital L some don't, not sure if it matters.

Reply:
Arbor only has official support for Linux and OS X, so we don't consider Windows. This is an easy one to fix when we do support Windows.
FWIW, from what I can tell, case sensitive path names are turned off by default in Windows, and have to be turned on for individual paths.